Content.Server/Medical/DefibrillatorSystem.cs

```diff
@@ -144,13 +154,13 @@ public bool TryStartZap(EntityUid uid, EntityUid target, EntityUid user, Defibri
 
         return _doAfter.TryStartDoAfter(new DoAfterArgs(EntityManager, user, component.DoAfterDuration, new DefibrillatorZapDoAfterEvent(),
             uid, target, uid)
-            {
-                BlockDuplicate = true,
-                BreakOnHandChange = true,
-                NeedHand = true,
-                BreakOnMove = !component.AllowDoAfterMovement,
-                DuplicateCondition = DuplicateConditions.SameEvent,
-            });
+        {
+            BlockDuplicate = true,
+            BreakOnHandChange = true,
+            NeedHand = true,
+            BreakOnMove = !component.AllowDoAfterMovement,
+            DuplicateCondition = DuplicateConditions.SameEvent,
+        });
```

Comment:
We usually leave upstream files formatting unchanged for less merge conflicts
```suggestion
            {
                BlockDuplicate = true,
                BreakOnHandChange = true,
                NeedHand = true,
                BreakOnMove = !component.AllowDoAfterMovement,
                DuplicateCondition = DuplicateConditions.SameEvent,
            });
```